### DISCUSSION TITLE
Comment marche la method pushEvent ?

### DISCUSSION THREAD

**Discussion by Jad31** [TempoDev/Arcade-Toulouse#64] [Questions]
La question est dans le titre, et je n'ai pas très bien compris également le lifecycle des events en gros.
Je ne comprend pas comment faire communiquer `push_event()` et les methods `onKeyEvent` et `onMouseEvent()`

**Comment by nasso** [comment]
dynamic_cast ! c'est ce qui était prévu à l'origine avec un getType il me semble, même chose mais avec un dynamic_cast
